main.js

```diff
@@ -173,6 +178,8 @@ var start_local_server = function() {
       serverRunning = false;
     });
     subpy.unref();
+  } else {
+    mainWindow.webContents.executeJavaScript("console.log('Unable to find OpenBazaar-Server at: '" + serverPath + "')");
```

Comment:
Before doing anything on the mainWindow instance, I think a check needs to be done to see if it's not null, eg:

```
mainWindow && mainWindow.mainWindow.webContents.executeJavaScript();
```

When the browser is closed, mainWindow is set to null and on Mac the app will remain open. So, if anything executes (e.g. the auto-updater callbacks) that tries to call something on a mainWindow instance that is actually null, we'll get an exception (perhaps a crash as well).